external/storm-blobstore-migration/src/main/java/org/apache/storm/blobstore/ListHDFS.java

```diff
@@ -0,0 +1,52 @@
+package org.apache.storm.blobstore;
+
+import java.util.Map;
+
+import javax.security.auth.Subject;
+
+import org.apache.storm.Config;
+import org.apache.storm.blobstore.ClientBlobStore;
+import org.apache.storm.hdfs.blobstore.HdfsBlobStore;
+import org.apache.storm.hdfs.blobstore.HdfsClientBlobStore;
+import org.apache.storm.utils.Utils;
+
+public class ListHDFS {
+    
+    public static void main(String[] args) throws Exception {
```

Comment:
Could you file a follow on JIRA to migrate this to be a part of the storm admin command?

Reply:
[STORM-2763](https://issues.apache.org/jira/browse/STORM-2763)